var/spack/repos/builtin/packages/thirdorder/package.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyThirdorder(PythonPackage):
```

Comment:
The unit tests are failing because the class name is PyThirdorder but the package name is thirdorder. If you change the filename to py-thirdorder that should fix it.

Reply:
done